### PR TITLE
Fix/unstack image upload modal

### PR DESCRIPTION
**Pull request by kwajiehao**
## Overview

This PR resolves a litany of UI issues:
- #161: Shift `Heading` button in markdown editor to the front of the toolbar
- #163: Unstack media modals so that there is only one modal open at a time
- #164: Reposition action buttons (save or delete actions) to the bottom right hand corner
- Refactor to subsume the `EditResourcePage` component into `EditPage` since there is a lot of duplicated code

## Screenshots

Issue #163 
_Before_
Screenshot 2020-10-20 at 4 51 15 PM

_After_
Screenshot 2020-10-20 at 4 50 43 PM

Issue #164 
_Before_
Screenshot 2020-10-20 at 4 52 19 PM
Screenshot 2020-10-20 at 4 52 38 PM

_After_
Screenshot 2020-10-20 at 4 53 02 PM
Screenshot 2020-10-20 at 4 53 17 PM

### CODE REVIEW

File: src/layouts/EditPage.jsx [hunk 1]
```diff
@@ -133,20 +147,53 @@ export default class EditPage extends Component {
...
+      },
+    });
+  }
+
+  readFileToStageUpload = async (event) => {
+    const fileReader = new FileReader();
+    const fileName = event.target.files[0].name;
+    fileReader.onload = (() => {
+      /** Github only requires the content of the image
+         * imgReader returns  `data:application/pdf;base64, {fileContent}`
```
Comment (alexanderleegs) [inline src/layouts/EditPage.jsx:168]: nit: fileReader

**Review by alexanderleegs — CHANGES_REQUESTED** [review]
Left a minor comment.

Also currently, if the user is in the image upload modal but changes his mind, the entire modal is closed - we could possibly bring up the image select modal again, any thoughts on this? I'm open to leaving it as is also.

File: src/layouts/EditPage.jsx [hunk 2]
```diff
@@ -133,20 +147,53 @@ export default class EditPage extends Component {
...
+        path: `${baseFolder}%2F${fileName}`,
+        content: fileData,
+        fileName,
+      },
+    });
+  }
+
+  readFileToStageUpload = async (event) => {
+    const fileReader = new FileReader();
+    const fileName = event.target.files[0].name;
```
Comment (kwajiehao) [inline src/layouts/EditPage.jsx:165]: test

Comment (kwajiehao) [inline src/layouts/EditPage.jsx:165]: test

File: src/layouts/EditPage.jsx (hunk 1 above)
Comment (kwajiehao) [reply src/layouts/EditPage.jsx]: fixed with [bc6bb97](https://github.com/isomerpages/isomercms-frontend/pull/169/commits/bc6bb97c5bb72f46bf54dd708301df63461f14ff)

**Comment by kwajiehao** [comment]
> if the user is in the image upload modal but changes his mind, the entire modal is closed - we could possibly bring up the image select modal again, any thoughts on this? I'm open to leaving it as is also.

Re: this, I'm inclined to leave it as it is for now, and change it if the designers think it's a problem, since it's a pretty easy fix

**Review by alexanderleegs — APPROVED** [review]
lgtm

**Pull request merged**